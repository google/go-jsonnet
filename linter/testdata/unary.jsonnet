[
    ~false,
    -"42",
    +"42",
    !42,
]
