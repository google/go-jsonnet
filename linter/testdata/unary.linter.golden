testdata/unary:2:5-11 Operand is not a number, it is assumed to be a bool

    ~false,


testdata/unary:3:5-10 Operand is not a number, it is assumed to be a string

    -"42",


testdata/unary:4:5-10 Operand is not a number, it is assumed to be a string

    +"42",


testdata/unary:5:5-8 Operand is not a boolean, it is assumed to be a number

    !42,


